### DISCUSSION TITLE
User-Level Permissions for Project Editing

### DISCUSSION THREAD

**Discussion by mauroalexandre** [frangoteam/FUXA#1342] [Ideas]
It would be interesting to change the project editing permission so that it is on a user level rather than a project level. What do you guys think of this feature? I could try making this change and submit a pull request.

In cases where FUXA is running behind a web server, the /editor route does not work, since it is an Angular route. This would solve the problem, and I also believe that user-level editing control allows for greater organization.

What's your opinion?

**Comment by unocelli** [comment]
Hi, this feature is a big challenge:
- can a user create his views and decide who can see them?
- if you can create views, you must also have access to the layout settings and charts settings section.
- can a user edit the views of others?

**Reply by mauroalexandre** [reply]
@unocelli 

I believe in keeping things simple for now; the permission role would only grant access to /editor. Currently, it's defined at the project level, but it would be interesting to have it defined at the user level.

Like this

![image](https://github.com/user-attachments/assets/3d08b714-d0d0-4896-8cb6-32d4a1dbc8a2)

**Reply by unocelli** [reply]
Yes, I'm agree, could be a beginning
